spec/support/fixtures/bookshelf/config/environment.rb

```diff
@@ -28,6 +28,7 @@ module Admin
       manifest         "assets.json"
       prefix           "/assets/admin"
       compile          true
+      nested           true
```

Comment:
These values are needed for `precompile_spec.rb:81`. Alternatively, we could remove the `style/` prefix there

Reply:
Please remember that because of global state, it's preferable to run integration specs one by one, instead of Rake/RSpec defaults. See https://github.com/hanami/controller/blob/master/script/test#L43-L50

Reply:
That wasn't on my radar. I'll update the script to do that 🙂

Reply:
@jodosha Are you thinking the intermittent failures (mostly in compiler_spec.rb) are a result of sharing global state? I updated `script/ci` to run each test individually. After a few runs locally, everything is finally looking good 😅 No flakes yet!

Also, since the config referenced in ^ this diff is duplicated, do you have a preference for whether this should be DRY?

Reply:
@landongrindheim 
> Are you thinking the intermittent failures (mostly in compiler_spec.rb) are a result of sharing global state?

Yes, that's why we're moving away from it.

> Also, since the config referenced in ^ this diff is duplicated, do you have a preference for whether this should be DRY?

Please don't. For maintenance purposes, it's better to not DRY too much testing code.

Reply:
@landongrindheim `nested` is `false` by default, we should leave it so, **unless** the `Admin` application needs this setting to be `true` because we wanted to test the behavior of this feature for this specific app.

Reply:
This behavior is being tested, at least implicitly, in precompile_spec.rb. Would you prefer the spec to be updated so that style/style.css becomes style.css, which would allow us to rely on the default config? Or would you prefer to continue exercising the nesting?

Reply:
@landongrindheim It's good to have a mix of both `nested true` (like in this case), and `nested false` like in the other two cases. 👍